File: TRNMNT.Core/Model/Round/MatchResultModel.cs
using System;
using TRNMNT.Core.Enum;

namespace TRNMNT.Core.Model.Round
{
    public class MatchResultModel
    {
        public Guid MatchId { get; set; }
        public Guid WinnerParticipantId { get; set; }
        public int CompleteTime { get; set; }
        public MatchResultTypeEnum RoundResultType { get; set; }
        public SubmissionTypeEnum SubmissionType { get; set; }
        public int AParticipantPoints { get; set; }
        public int AParticipantAdvantages { get; set; }
        public int AParticipantPenalties { get; set; }
        public int BParticipantPoints { get; set; }
        public int BParticipantAdvantages { get; set; }
        public int BParticipantPenalties { get; set; }
    }
}

File: TRNMNT/ClientApp/app/core/model/match-result.model.ts
export class MatchResultModel {
    matchId: AAGUID;
    winnerParticipantId: AAGUID;
    aParticipantPoints: number;
    aParticipantAdvantages: number;
    aParticipantPenalties: number;
    bParticipantPoints: number;
    bParticipantAdvantages: number;
    bParticipantPenalties: number;
    submissionType : number;
    roundResultType : number;
    completeTime: number;
}